cli/aiconfig-editor/pages/api/aiconfig/load.ts

```diff
@@ -24,10 +29,32 @@ export default function handler(
     return res.status(500).json({ error: "No path provided" });
   }
 
-  // TODO: Needed to add node-fetch to the dependencies of aiconfig-editor for this to work
-  // This should be fixed up in library though?
   // TODO: load should probably be async?
   const aiconfig = AIConfigRuntime.load(body.path);
 
-  res.status(200).json({ aiconfig });
+  // Refine outputs for client-side rendering. We only care about displaying (and deleting)
+  // outputs directly from the editor
+  const clientAIConfig: ClientAIConfig = {
+    ...aiconfig,
+    prompts: aiconfig.prompts.map((prompt) => ({
+      ...prompt,
+      outputs: prompt.outputs?.map((output) => {
+        if (output.output_type === "execute_result") {
```

Comment:
I'm guessing this is where we'll also add handling for error output types?

Reply:
Error types should already be matching the Error type and we can get the error value/trace from that without special handling